Source/Resources/en.lproj/Localizable.strings

```diff
@@ -1,40 +1,41 @@
-"missingBillingFormFullName" = "Missing Name";
-"missingBillingFormAddressLine1" = "Missing Address Line 1";
-"missingBillingFormAddressLine2" = "Missing Address Line 2";
-"missingBillingFormCity" = "Missing City";
-"missingBillingFormCounty" = "Missing County";
-"missingBillingFormPostcode" = "Missing Postcode";
-"missingBillingFormPhoneNumber" = "Missing Phone Number";
-"missingBillingFormCountry" = "Missing Country";
-"billingFormPhoneNumberHint" = "Phone Number";
-"billingAddressTitle" = "Billing address";
-"BillingFormCountryButtonTitle" = "Please select a country";
-"PaymentFormSummaryViewHintText" = "We need this information as an additional security measure to verify this card.";
-"AddBillingAddress" = "Add billing address";
+"Missing Name" = "Missing Name";
+"Missing Address Line 1" = "Missing Address Line 1";
+"Missing Address Line 2" = "Missing Address Line 2";
+"Missing City" = "Missing City";
+"Missing County" = "Missing County";
+"Missing Postcode" = "Missing Postcode";
+"Missing Phone Number" = "Missing Phone Number";
+"Missing Country" = "Missing Country";
+"Please select a country" = "Please select a country";
+"We need this information as an additional security measure to verify this card." = "We need this information as an additional security measure to verify this card.";
+"Add billing address" = "Add billing address";
 /* Accessory shown alongside input fields to denote that the input is not required for validation when completing request */
-"OptionalInputField" = "Optional";
+"Optional" = "Optional";
 /* Title of input section on the Payment Form where user is expected to input the Cardholder name as displayed on card */
-"CardholderInputTitle" = "Cardholder name";
-"ExpiryDate" = "Expiry date";
-"ExpiryDateFormat" = "Format is MM/YY";
-"ExpiryDatePlaceholder" = "_ _  / _ _";
-"ExpiryDateErrorMessageInvalid" = "Please enter a valid expiry date";
-"ExpiryDateErrorMessageInvalidInPast" = "Expiry date is in the past";
-"SecurityCode" = "Security code";
-"SecurityCodeHint" = "3 - 4 digit code on your card";
-"SecurityCodeErrorMessage" = "Please enter a valid security code";
-"PaymentHeaderTitle" = "Payment details";
-"PaymentHeaderSubtitle" = "Visa, Mastercard and American Express accepted.";
-"CardNumber" = "Card number";
-"CardNumberErrorMessage" = "Please enter a valid card number";
+"Cardholder name" = "Cardholder name";
```

Comment:
That comment just became reassigned to a different key pair making it confusing instead of helping. Please undo all changes to this file.